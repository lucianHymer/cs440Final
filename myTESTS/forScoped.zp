//simple.cmm 
//Input: None
//Output: 2017 
int main ()
{
  int sum;
  int i;

  i = 12;

  sum = 2016;
  for ( int i = 0; i < 3; i = i + 1 ) putn(i);
  putn (sum);
  putn (i);
  return 0;
}
