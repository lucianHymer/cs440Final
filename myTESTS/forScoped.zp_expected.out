     0 
     1 
     2 
  2016 
    12 
